packages/vscode/src/telemetryReporter.ts

```diff
@@ -1,14 +1,22 @@
 import { check } from 'checkpoint-client'
 import { Disposable, workspace } from 'vscode'
 import { getProjectHash } from './hashes'
+
+type TelemetryLevel = 'off' | 'crash' | 'error' | 'all' | undefined
+
 export default class TelemetryReporter {
   private userOptIn = false
   private readonly configListener: Disposable
 
-  private static TELEMETRY_CONFIG_ID = 'telemetry'
-  private static TELEMETRY_CONFIG_ENABLED_ID = 'enableTelemetry'
+  private static TELEMETRY_SECTION_ID = 'telemetry'
+  private static TELEMETRY_SETTING_ID = 'telemetry.telemetryLevel'
+  // Deprecated since https://code.visualstudio.com/updates/v1_61#_telemetry-settings
+  private static TELEMETRY_OLD_SETTING_ID = 'telemetry.enableTelemetry'
```

Comment:
Previously, it was just `'enableTelemetry'`, so this is also part of the fix, iiuc?